### PR TITLE
Increase default command timeout on operations against SQL Server master

### PR DESCRIPTION
**Pull request by natemcmaster**
Compensate for slowness in creating a database on SQL Server.

### CODE REVIEW

File: test/EntityFramework.MicrosoftSqlServer.FunctionalTests/Utilities/SqlServerTestStore.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ namespace Microsoft.Data.Entity.SqlServer.FunctionalTests
 {
     public class SqlServerTestStore : RelationalTestStore
     {
-        public const int CommandTimeout = 30;
+        public const int CommandTimeout = 90;
```
Comment (smitpatel) [inline test/EntityFramework.MicrosoftSqlServer.FunctionalTests/Utilities/SqlServerTestStore.cs]: Why 2 different values as 60 & 90?

Comment (natemcmaster) [inline test/EntityFramework.MicrosoftSqlServer.FunctionalTests/Utilities/SqlServerTestStore.cs]: I make SqlServerTestStore more lenient due to the number of scratch database it creates/deletes.

File: src/EntityFramework.MicrosoftSqlServer/Storage/Internal/SqlServerConnection.cs [hunk 2]
```diff
@@ -11,6 +11,9 @@ namespace Microsoft.Data.Entity.Storage.Internal
 {
     public class SqlServerConnection : RelationalConnection, ISqlServerConnection
     {
+        // Compenstate for slow SQL Server database creation
```
Comment (lajones) [inline src/EntityFramework.MicrosoftSqlServer/Storage/Internal/SqlServerConnection.cs]: Typo: Compenstate.

**Comment by lajones** [comment]
Looks good. :shipit:

**Pull request merged**